screenpipe-server/src/core.rs

```diff
@@ -32,28 +33,27 @@ pub async fn start_continuous_recording(
     friend_wearable_uid: Option<String>,
     monitor_id: u32,
     use_pii_removal: bool,
+    vad_engine: CliVadEngine,
 ) -> Result<()> {
-    let (whisper_sender, whisper_receiver, whisper_shutdown_flag) = if audio_disabled {
```

Comment:
??

Reply:
I considered your PR(#250) While making changes. But i'll add shutdown flag here also and push the code